src/canmatrix/canmatrix.py

```diff
@@ -1174,8 +1352,13 @@ def updateEcuList(self):
                     self.addEcu(BoardUnit(ecu))
 
     def renameFrame(self, old, newName):
+        """Rename Frame.
+
+        :param Frame or str old: Old Frame instance or name or part of the name with '*' at the beginning or the end.
+        :param str newName: new Frame name, suffix or prefix
+        """
         if type(old).__name__ == 'instance':
-            oldName = old.name
+            old = old.name
```

Comment:
+ you found a bug!

Reply:
What has a class name of (lower-case) `instance` anyways?